packages/navigator/src/browser/navigator-widget.tsx

```diff
@@ -213,8 +213,10 @@ export class FileNavigatorWidget extends FileTreeWidget {
     }
 
     protected handleClickEvent(node: TreeNode | undefined, event: React.MouseEvent<HTMLElement>): void {
-        if (node && this.corePreferences['list.openMode'] === 'singleClick') {
-            this.model.previewNode(node);
+        if (!event.shiftKey && !event.ctrlKey) {
```

Comment:
`ctrlKey` is OS specific: `IF isOSX THEN check event.metaKey ELSE check event.ctrlKey`

Reply:
I'v add osx compatible code, but I don't hava Mac to test if it is work properly